docs/guide/getstarted/boilerplate.md

```diff
@@ -11,32 +11,48 @@ Over the time our community has developed a bunch of boilerplate projects that c
 
 ## [amiya-pattnaik/webdriverIO-with-cucumberBDD](https://github.com/amiya-pattnaik/webdriverIO-with-cucumberBDD)
 
-- Framework: Cucumber (v 3.0.0)
+- Framework: Cucumber (v4.x)
```

Comment:
WebdriverIO doesn't support Cucumber v4 yet, let's remove the version number here